src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -66,17 +86,34 @@
 #include "native_client/src/trusted/service_runtime/win/debug_exception_handler.h"
 
 #if NACL_WINDOWS
-#include "native_client/src/trusted/service_runtime/win/debug_exception_handler.h"
-#include "native_client/src/shared/platform/win/xlate_system_error.h"
+#  include "native_client/src/trusted/service_runtime/win/debug_exception_handler.h"
+#  include "native_client/src/shared/platform/win/xlate_system_error.h"
 #endif
 
 #include "native_client/src/trusted/validator/ncvalidate.h"
 #include "native_client/src/trusted/validator/validation_metadata.h"
+// yiwen
+#include "native_client/src/trusted/service_runtime/env_cleanser.h"
+#include "native_client/src/trusted/service_runtime/lind_syscalls.h"
+#include "native_client/src/trusted/service_runtime/nacl_all_modules.h"
+#include "native_client/src/trusted/service_runtime/nacl_app.h"
+#include "native_client/src/trusted/service_runtime/load_file.h"
 
 struct NaClDescQuotaInterface;
 
 static size_t const kdefault_io_buffer_bytes_to_log = 64;
 
+/*
+ * yiwen: my data for the in-process-pipe
+ *
+ * char pipe_buffer[16*4096];
+ * int pipe_mutex;
+ *
+ * 0: pipe is empty, ready to write, cannot read; 1: pipe is full, ready to read, cannot write.
+ * at initialization, it should be set to 0.
+ */
+/* char *buffer_ptr; */
```

Comment:
this code is commented out. Do you need it?

Reply:
Nope, removing.